Missing accessibility (a11y) tests

* Easy first static check would be to add "plugin:jsx-a11y/recommended" to your eslint config. Ideally should enable it for all console.redhat.com projects

I shortly tried it out and the situation is actually really good:

```
[kkoukiou@sequioa image-builder-frontend]$ npm run lint 

> image-builder@1.1.0 lint
> npm-run-all lint:*


> image-builder@1.1.0 lint:js
> eslint config src


/home/kkoukiou/repos/image-builder-frontend/src/Components/CreateImageWizard/formComponents/ReviewStep.js
  151:11  error  The autoFocus prop should not be used, as it can reduce usability and accessibility for users              jsx-a11y/no-autofocus
  157:19  error  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text
  189:19  error  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text
  217:19  error  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text

/home/kkoukiou/repos/image-builder-frontend/src/Components/CreateImageWizard/formComponents/TargetEnvironment.js
  65:17  error  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text
  81:17  error  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text
  99:17  error  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text

✖ 7 problems (7 errors, 0 warnings)

ERROR: "lint:js" exited with 1.

```
* For more advanced tests the suggested tool to use is https://github.com/patternfly/patternfly-a11y, which is based on axe. This however implies browser based testings, which is currently missing in your tests, as you use jest tests on a fake DOM.

--- a/.eslintrc.yml
+++ b/.eslintrc.yml
@@ -1,4 +1,5 @@
 extends: [
+    "plugin:jsx-a11y/recommended",
     "@redhat-cloud-services/eslint-config-redhat-cloud-services",
     "plugin:react/recommended",
     "plugin:react-hooks/recommended",
@@ -39,6 +40,7 @@ rules:
   no-console: 2
   eqeqeq: error
   array-callback-return: warn
+  jsx-a11y/no-autofocus: warn
 overrides:
   - files: "**/*.ts?(x)"
     parser: "@typescript-eslint/parser"

--- a/package.json
+++ b/package.json
@@ -81,6 +81,7 @@
     "eslint": "8.46.0",
     "eslint-plugin-import": "2.28.1",
     "eslint-plugin-jest-dom": "5.1.0",
+    "eslint-plugin-jsx-a11y": "6.8.0",
     "eslint-plugin-react": "7.33.0",
     "eslint-plugin-react-hooks": "4.6.0",
     "eslint-plugin-testing-library": "6.1.0",

